.github/workflows/ci.yaml

```
@@ -0,0 +1,78 @@
+      - main
+
+jobs:
+  build:
+    name: CI - Full repo
+    runs-on: ubuntu-latest
+    env:
+      TURBO_API: ${{ vars.TURBO_API }}
+      TURBO_TEAM: ${{ vars.TURBO_TEAM }}
+      TURBO_TOKEN: ${{ secrets.TURBO_TOKEN }}
+      IMAGE_NAME: ghcr.io/${{ github.repository }}-api
+      VERSION=${{ github.sha }}
```

actionlint (1.7.4)

could not parse as YAML: yaml: line 17: could not find expected ':'

(syntax-check)